src/vector3.h

```diff
@@ -150,7 +157,7 @@ class alignas(sizeof(T)) vector3 {
 		}
 	}
 
-	void Print() const { printf("v(%f,%f,%f)\n", x, y, z); }
+	inline void Print() const { printf("v(%f,%f,%f)\n", x, y, z); }
```

Comment:
Do we... need this? I'd much rather drop the header-level dependency on stdio.h.

Reply:
I checked and it's not used, so I'll remove it and the ones from vector2, matrix3x3, and matrix4x4 as well!